DirectEmployers/MyJobs
# Modify models to facilitate partner assessments [PD-3473]

Also adds initial feature flag support

## What's this PR do?
Allows us to start work on partner assessments
## Screenshots (if appropriate)
Feature flag disabled (or nonexistent)
![secure my jobs-prm-view-records-edit-id 2 partner 2 page 1 1](https://user-images.githubusercontent.com/3379265/28173096-694ef1a4-67bb-11e7-9cc3-a20cf6ea05e6.png)

Feature flag enabled - this actually features horizontal radio buttons but the base bootstrap style we load makes them display:block
![secure my jobs-prm-view-records-edit-id 2 partner 2 page 1](https://user-images.githubusercontent.com/3379265/28173103-6ebf1aec-67bb-11e7-816b-42353ee9880f.png)
## Where should the reviewer start?

## How should this be manually tested?
Check out the "edit communication record" form and verify that it doesn't have inputs for score or review. The fields currently don't show up on the NUO version, just the one in PRM.

Go to your admin and find the new section named "Waffle" and click on the "add" button next to "flags". Name your new flag "partner assessment" and enable it for superusers, staff, or authenticated. Your dev user is probably all of these so which you pick doesn't matter.

Go back to the "edit communication record" form and, like magic, there are new fields for you to fill out.
## Any background context you want to provide?
